CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 - Update the contribution guideline to note that updating the changelog is required. (#5586)
+- Remove Pester as a module include with the PowerShell Packages.
+  In the future, you should be able to add it by running `Install-Module Pester`. (#5623, #5631)
```

Comment:
Should we have this as a sub bullet point to make it clear it's part of the previous sentence.  I think typically we've had changes on a single line, but would not want long lines either.

Reply:
Shouldn't `Start-PSBootstrap` check for Pester and just install it (CurrentUser scope) if it isn't on the user's system?

Reply:
Why is not a single bullet point adequate?

Reply:
`start-psbootstrap` is for the development environment, this is referring to the product.